source/Octopus.Tentacle.Tests/Capabilities/CapabilitiesServiceV2Fixture.cs

```diff
@@ -10,7 +10,7 @@ public class CapabilitiesServiceV2Fixture
         public void CapabilitiesAreReturned()
         {
             var capabilities = new CapabilitiesServiceV2()
```

Comment:
We don't need an await here?

Reply:
We do, I just hadn't finished the work off from our pairing session. It should be there now.